lib/registration.pm

```diff
@@ -115,6 +116,20 @@ sub add_suseconnect_product {
     assert_script_run("SUSEConnect -p $name/$version/$arch $params");
 }
 
+=head2 remove_suseconnect_product
+
+    remove_suseconnect_product($name, [$version, [$arch, [$params]]]);
+
+Wrapper for SUSEConnect -d $name.
+=cut
+sub remove_suseconnect_product {
+    my ($name, $version, $arch, $params) = @_;
+    $version //= scc_version();
+    $arch    //= get_required_var('ARCH');
+    $params  //= '';
+    assert_script_run("SUSEConnect -p $name/$version/$arch $params");
```

Comment:
This option should be `-d`?

Reply:
Yes, that's totally correct. I'll provide a fix.